routes/queries.js

```diff
@@ -0,0 +1,68 @@
+const db = require('../db')
+
+module.exports = {
+
+	// Adds user to users db
+	addUser: async function(firstname, lastname, email, secret) {
+		const insertUser = {
+			text: 'INSERT INTO users(firstname, lastname, email, secret) VALUES ($1, $2, $3, $4) RETURNING id',
+			values: [firstname.toLowerCase(), lastname.toLowerCase(), email.toLowerCase(), secret]
+		}
+		return db.query(insertUser)
+	},
+		
+	// Checks for duplicate users with same email
+	// Goes through database and counts where email matches. Returns true if user is alreayd registered
+	emailAlreadyRegistered: async function (email) {
+		const checkdup = {
+			text: 'SELECT COUNT(email) FROM users WHERE email = $1',
+			values: [email.toLowerCase()]
+		}
+		const result = await db.query(checkdup)
+		return (result.rows[0].count > 0)
+	},
+
+	// Get user data from secret
+	getUserData: async function (secret) {
```

Comment:
[nit] rename to `getUserBySecret` to clarify the parameter, differentiating it from a potential `getUserByEmail` function